saleor/graphql/attribute/tests/mutations/test_attribute_create.py

```diff
@@ -1255,3 +1255,43 @@ def test_create_attribute_and_attribute_values_errors(
     assert errors[0]["field"] == "values"
     assert errors[0]["message"] == error_msg
     assert errors[0]["code"] == error_code.name
+
+
+def test_create_attribute_similar_names(
```

Comment:
Could we also add a similar one for `update_attribute` mutation, when the value with `15` already exists and we are adding a new `1.5` value?